shared/fs/folder/offline.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as Kb from '../../common-adapters'
+import React from 'react'
+import * as Styles from '../../styles/index'
+import * as Types from '../../constants/types/fs'
+import TopBar from '../top-bar'
+
+type Props = {|
+  path: Types.Path,
+|}
+
+const OfflineFolder = (props: Props) => (
+  <Kb.Box2 direction="vertical" style={styles.contentContainer} fullWidth={true} alignItems={'stretch'}>
+    <Kb.BoxGrow>
```

Comment:
It doesn't seem like this `BoxGrow` is necessary here. We need it in the folder view because it needs to know the height and BoxGrow makes a positioned div and gives that. In this case I think `flex: 1` would just work. Have you tried removing `<Kb.BoxGrow>`?